tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "declaration": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
`moduleResolution: "bundler"` is new in Typescript 5. Roughly, it lets use the `"exports"` field in `package.json`, which gives us better control over what is exported from a package. 
- see https://nodejs.org/api/packages.html#package-entry-points for more on `exports`
- https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#moduleresolution-bundler for more. for more on `"moduleResolution": "bundler"`.

*Additionally, `"moduleResolution": "node"` is somewhat discouraged, and is a deprecation candidate for Typescript 6. Though...I'm sure it will be around for a couple more years.*